service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/ActionCaseManagementOrderController.java

```diff
@@ -66,13 +66,10 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToStart(@RequestBody Call
                 .build();
         }
 
-        caseDetails.getData().putAll(caseManagementOrderService
-            .extractMapFieldsFromCaseManagementOrder(caseManagementOrder));
-
         draftCMOService.prepareCustomDirections(caseDetails, caseManagementOrder);
 
+        caseDetails.getData().putAll(caseManagementOrder.getCCDFields());
```

Comment:
Previous implementation had NPE guard inside. But I guess it's not possible to have null here in Action?